docs/pages/guides/tutorials/deploying-php-app-with-redis.md

```diff
@@ -0,0 +1,108 @@
+
+
+### Steps
+
+#### Download the Kubernetes Configuration Files
+
+To get started we will need some yaml files that will contain the configurations for the pods and services that will run on Meshery. You can download them to see what the look like.
+1. [redis-leader-deployment.yaml](https://k8s.io/examples/application/guestbook/redis-leader-deployment.yaml)
+2. [redis-leader-service.yaml](https://k8s.io/examples/application/guestbook/redis-leader-service.yaml)
+3. [redis-follower-deployment.yaml](https://k8s.io/examples/application/guestbook/redis-follower-deployment.yaml)
+4. [redis-follower-service.yaml](https://k8s.io/examples/application/guestbook/redis-follower-service.yaml)
+5. [frontend-deployment.yaml](https://k8s.io/examples/application/guestbook/frontend-deployment.yaml)
+6. [frontend-service.yaml](https://k8s.io/examples/application/guestbook/frontend-service.yaml)
+
+These YAML files contain the Service definitions and Deployment configurations for the PHP app with Redis.
+
+#### Accessing Meshery Playground
+
+1. Log in to the [Meshery Playground](https://meshery.layer5.io/) using your credentials. On successful login, you should be at the dashboard. Press the **X** on the _Where do you want to start?_ popup to close it (if required).
+
+2. Click **Explore** in the Cloud Native Playground tile to navigate to _MeshMap_
+
+
+#### Import the Files to Meshery Playground
+
+1. In the left sidebar, click on the upward arrow symbol(import icon) to import the designs into Meshery.
+
+2. In the modal that appears:
+   - Enter a name for your design in the "Design File Name" field (e.g.`redis-leader-deployment`).
+
+   - Select `Kubernetes Manifest` from the "Design Type" dropdown menu.
+
+   - Choose `URL Import` for the upload method, and input the URL.
+
+   - Then, click on `Import`
+
+        ![Import redis-leader-deployment](./screenshots/redis-leader-deployment.png)
+
+3. Now, follow the same steps (1-2) to import the rest of the files.
+
+4. Under the “Designs” tab, you will see that we have successfully imported the manifest as designs, then you can drag and drop them in the canvas. This will "Merge" all the designs since it's all just one application
+    ![merging all designs](./screenshots/app-canvas.png)
+
+#### Scaling Deployments
+You might be wondering how you would go about scaling your microservice. Here we will, explore how to scale individual components of the microservices application by adjusting the replicas in the Deployment configuration.
+1. 
+
+
+
+#### 4. **Updating Deployments:**
+   - Learn how to update the microservices application by rolling out changes to the Deployment.
+
+\```bash
+kubectl set image deployment/<deployment-name> <container-name>=new-image:tag
+\```
+
+#### 6. **Monitoring and Troubleshooting Deployments:**
+  - Utilize Meshery Playground to monitor the status of Deployments and troubleshoot common issues.
```

Comment:
Probably a good idea to have a screenshot where to monitor or troubleshoot. Or if that's beyond the scope of the article remove this line.